Tools/LyTestTools/tests/integ/__init__.py

```diff
@@ -1,6 +1,10 @@
-"""
+r"""
```

Comment:
why are you prepending an r to this docstring block? I'm guessing this is something I don't know.

Reply:
It makes it a raw string. In this case it make sure it contains literal backslashes instead of using backslash as an escape character. I wanted the string below to be copyable from the file into a Windows CLI, while also not making garbled text in the docstring.